packages/frontend/src/components/MapSearchResultList.tsx

```diff
@@ -17,15 +18,30 @@ export const MapSearchResultList = () => {
         if (searchResult === null || !searchResult.length) {
             return;
         }
-        if (halfWay != undefined) {
-            scroll.current.scrollTo(0, halfWay.current.offsetTop);
+        if (halfWay != undefined && halfWay.current != undefined) {
+            const scrollElm = scroll.current as HTMLDivElement;
+            scrollElm.scrollTo(0, halfWay.current.offsetTop);
+            scrollElm.addEventListener("scroll", (e) => {
```

Comment:
Das gibt glaube ich nen event listener memory leak, weil der immer wieder geadded wird und niemals removed wird. macht man in class components normalerweise mit dem componentWillUnmount hook.

Reply:
https://stackoverflow.com/a/12528067
Bei neu zeichnen wird das Element doch ohne Referenzen gelöscht, oder?

Reply:
Das element existiert immernoch in-mermoy, wenn es wieder in die DOM kommt haengst da ja dann nochmal ein listener dran und dann nochmal etc.? Wenn das element geloescht wird und da noch listener dran haengen, kann es nicht garbage collected werden.

Reply:
nehme das onscroll, das kann es nur einmal geben